File: ELSA.Web.Client/src/models/auth/login.model.ts
export class LoginModel {
    constructor(username: string, password: string, keepMeLogIn: boolean) {
        this.username = username;
        this.password = password;
        this.keepMeLogIn = keepMeLogIn;
    }
    username!: string;
    password!: string;
    keepMeLogIn: boolean = false;
}

File: ELSA.API/ELSA.CodeChallenge.WebAPI/ELSA.CodeChallenge.WebAPI/Models/Authentication/LoginModel.cs
namespace ELSA.WebAPI.Models.Authentication
{
    /// <summary>
    /// Login information
    /// </summary>
    public class LoginModel
    {
        /// <summary>
        /// The user name or email
        /// </summary>
        public string Username { get; set; }
        /// <summary>
        /// The password
        /// </summary>
        public string Password { get; set; }
        /// <summary>
        /// Use for refresh token
        /// </summary>
        public bool KeepMeLogIn { get; set; }
    }
}

